scripts/scripts.js

```diff
@@ -15,34 +14,6 @@ import {
 
 const LCP_BLOCKS = []; // add your LCP blocks to the list
 
-/**
- * Builds hero block and prepends to main in a new section.
- * @param {Element} main The container element
- */
-function buildHeroBlock(main) {
```

Comment:
Could you leave the auto-blocking in place for the simple heros? Ideally the hero on a page like this could still benefit from auto blocking: https://careers.audemarspiguet.com/en/offer/995135/